blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/AbstractRunImpl.java

```diff
@@ -83,7 +82,7 @@ public BlueRunState getStateObj() {
         } else if(!run.isLogUpdated()){
             return BlueRunState.FINISHED;
         } else {
-            return BlueRunState.QUEUED;
```

Comment:
I thought QUEUED is right state to return, why change to RUNNING? Also if RUNNING is really the right state to return the conditions could be further simplified.